tests/js/README.md

```diff
@@ -0,0 +1,32 @@
+# Winter JS Framework Testing Suite
+
+The files in this directory form the testing suite for the new Winter JS Framework (`framework-next`). You must install
+all Node dependencies in order to run the testing suite:
+
+```bash
+cd tests/js
+npm install
+```
+
+You can then run the tests by using the following command:
+
+```bash
+npm run test
+```
+
+Please note that the tests are run against the "built" versions of Winter JS Framework and modules, in order to test
+the exact same functionality that would be delivered to the end user. We do this by leveraging the `JSDOM` library to
+simulate an entire HTML document.
+
+Therefore, you must compile a new build if you make any changes and wish to run the tests:
+
+```bash
+php artisan mix:compile --package framework
```

Comment:
More of a comment for the `mix:compile` and `winter:test` commands, but we should have the same alias for both of them shouldn't we? Or does it make more sense for this to be `--package snowboard`?

Reply:
Probably should be "snowboard" to make it clear what is being compiled - it'll also be less likely a name to be used anywhere else.